packages/features/kbar/Kbar.tsx

```diff
@@ -7,14 +7,17 @@ import {
   KBarSearch,
   useKBar,
   useMatches,
+  useRegisterActions,
 } from "kbar";
+import type { Action } from "kbar";
 import { useRouter } from "next/router";
 import { useMemo } from "react";
 
 import { appStoreMetadata } from "@calcom/app-store/appStoreMetaData";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { isMac } from "@calcom/lib/isMac";
-import { RouterOutputs, trpc } from "@calcom/trpc/react";
```

Comment:
Is there a reason why this import is broken into two? 🤔

Reply:
Due to our recent changes to eslint. we have to explicitly import as a type while importing a type.